packages/unmock-core/src/index.ts

```diff
@@ -71,21 +63,43 @@ export abstract class CorePackage implements IUnmockPackage {
     this.backend.reset();
   }
 
-  public get allowedHosts() {
+  // Allowd Hosts methods
+  public setAllowedHosts(urls: Array<string | RegExp> | string | RegExp): void {
+    this.whitelist = Array.isArray(urls) ? urls : [urls];
+    this.regexWhitelist = whitelistToRegex(this.whitelist);
+  }
+  public extendAllowedHosts(
+    urls: string | RegExp | Array<string | RegExp>,
+  ): void {
+    Array.isArray(urls)
+      ? this.whitelist.push(...urls)
+      : this.whitelist.push(urls);
+    this.regexWhitelist = whitelistToRegex(this.whitelist);
+  }
+  public getAllowedHosts() {
     return this.whitelist.map((url: string | RegExp) =>
       url instanceof RegExp ? url.source : url,
     );
   }
-  public set allowedHosts(urls: Array<string | RegExp>) {
-    this.whitelist = urls;
-    this.regexWhitelist = whitelistToRegex(this.whitelist);
-  }
   public isWhitelisted(host: string) {
-    if (this.regexWhitelist === undefined) {
-      return false;
-    }
     return this.regexWhitelist.filter(wl => wl.test(host)).length > 0;
   }
 
+  // Flaky mode methods
+  public flaky() {
+    this.isFlaky = true;
+  }
+  public nonFlaky() {
+    this.isFlaky = false;
+  }
+
+  // Use in production methods
+  public useInProduction() {
+    this.activeInProduction = true;
+  }
+  public useInDevelopment() {
+    this.activeInProduction = false;
```

Comment:
Not sure if `useInDevelopment` directly translates "do not use in production", to me that's more like "only use in development" 🤔  Maybe could also expose setting objects like `setting.on()` and `setting.off()` i.e. `unmock.someSetting.on()` but it does not really matter.